doc/library/config.txt

```diff
@@ -644,7 +644,8 @@ import theano and print the config variable, as in:
     <http://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/#options-for-steering-cuda-compilation>`_)
     mode for compiled cuda code which makes div and sqrt faster at the
     cost of precision.  This also disables support for denormal
-    numbers.
+    numbers. This can cause NaN. So if you have NaN and use this flag,
+    try to disable it.
```

Comment:
Can you add that information to `doc/tutorial/nan_tutorial.txt`? This would be a logical place.